traitsui/testing/tester/abstract_registry.py

```diff
@@ -0,0 +1,37 @@
+#  Copyright (c) 2005-2020, Enthought, Inc.
+#  All rights reserved.
+#
+#  This software is provided without warranty under the terms of the BSD
+#  license included in LICENSE.txt and may be redistributed only
+#  under the conditions described in the aforementioned license.  The license
+#  is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+#  Thanks for using Enthought open source!
+#
+
+import abc
+
+
+class AbstractRegistry(abc.ABC):
```

Comment:
Quick drive-by comment on the topic of "naming things is hard": this should probably be called `Abstract[something]Registry` (`AbstractInteractionRegistry`?) rather than `AbstractRegistry` (which is too generic a name).